docs/conf.py

```diff
@@ -32,11 +32,13 @@
 
 import os
 import sys
+from typing import Dict
```

Comment:
why, is this used somewhere?

Reply:
no, I have merged just the diffs from dlk/docs which is not correct since the first commit. fixed!
thank you for the review